jenkins/python-setup.sh

```diff
@@ -0,0 +1,14 @@
+_prefix="${1}"
+
+pip3 install --no-cache-dir --prefix="${_prefix}" -r ${progdir}/agent/requirements.txt -r ${progdir}/server/requirements.txt -r ${progdir}/agent/test-requirements.txt -r ${progdir}/server/test-requirements.txt
```

Comment:
If you installed all the python rpm dependencies, python-click, etc? Then why are you installing them again?

Reply:
Also why are we not using tox?

Reply:
> If you installed all the python rpm dependencies, python-click, etc? Then why are you installing them again?

We are not installing them again.  So first, we many module dependencies which are not entirely satisfied by RPMs. At the time of this writing, that would be `elasticsearch1`, `pyesbulk`, and `pidilfe` (`python-pidfile` is the PyPI module).  By using `pip3 install` a second time, in combination with `--current-env` for `tox`, we avoid a very lengthy install process for the Jenkins tests.  For more information, see 2695e8e2a436cb0eaaa2099f231144dffad0c227 (PR #1936).

Reply:
> Also why are we not using tox?

We are using `tox`, with the `--current-env`.  This is how we have tests running under 3 minutes.  Otherwise, we have much longer setup times for our tests.

How long does `tox` take to install modules in each environment when no modules are previously available?